@mortendybdal Can you please take a look at the linting errors on your PR?
```
  399:8   error    Expected JSDoc for 'target' but found 'json'  valid-jsdoc
  402:15  error    Missing space before function parentheses     space-before-function-paren
  415:30  error    Unexpected trailing comma                     comma-dangle
  420:20  error    There should be no space after '{'            object-curly-spacing
  420:42  error    There should be no space before '}'           object-curly-spacing
```

--- a/src/virtual-machine.js
+++ b/src/virtual-machine.js
@@ -396,10 +396,10 @@ class VirtualMachine extends EventEmitter {
 
     /**
      * Add a single sb3 sprite.
-     * @param {string} json JSON string representing the sprite/target.
+     * @param {string} target JSON string representing the sprite/target.
      * @returns {Promise} Promise that resolves after the sprite is added
      */
-    addSprite3(target) {
+    addSprite3 (target) {
         // Validate & parse
         if (typeof target !== 'string') {
             log.error('Failed to parse sprite. Non-string supplied to addSprite3.');
@@ -412,12 +412,12 @@ class VirtualMachine extends EventEmitter {
         }
 
         const jsonFormatted = {
-            targets: [target],
+            targets: [target]
         };
 
         return sb3
             .deserialize(jsonFormatted, this.runtime, null)
-            .then(({ targets, extensions }) => this.installTargets(targets, extensions, false));
+            .then(({targets, extensions}) => this.installTargets(targets, extensions, false));
     }
 
     /**

